ci/Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM ubuntu:20.04
```

Comment:
I prefer Debian/Devuan. Any specifics we need from Ubuntu? Probably we should start to use bookworm (Devuan hasn't released anything based on bookworm yet).

Reply:
I just see most of the people using ubuntu.